src/App/ComponentsDocumentation/components/Pagination/index.js

```diff
@@ -1,68 +1,111 @@
+        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={PaginationShowCasePanel} codeFigure/>
 
-const PaginationBullets = () => (
-    <>
-        <h2 id="pagination-bullets">Pagination bullets</h2>
-        <p>
-                If you want to show bullets instead of numbers you can add <CodeTags type="secondary" code=".pagination-bullets" /> to your <CodeTags type="primary" code={"<ul>"} />.
-        </p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <PaginationComponent type="bullets" items={paginationItems} arrows farArrows />
-        </ComponentPreview>
-    </>
+        <h3>Responsive design</h3>
+        <div className="slab slab-plain">
+            <nav className="pagination align-items-center no-pointer-events" role="navigation" aria-label="Pagination Navigation">
+                <button className="arrow-start d-block disabled">
+                    <i className="material-icons" aria-label="start"></i></button>
+                <button className="arrow-back disabled">
+                    <i className="material-icons" aria-label="back"></i>
+                </button>
+                <span className="mobile d-block font-weight-bold">Page 1 of 10</span>
+                <button className="arrow-forward">
+                    <i className="material-icons" aria-label="forward"></i>
+                </button>
+                <button className="arrow-end d-block">
+                    <i className="material-icons" aria-label="end"></i>
+                </button>
+            </nav>
+        </div>
+
+        <p>On smaller screens the page links are replaced with text showing the page information, still using arrows to go to next or previous page but with the option to go to the first or last page as well.</p>
+
+        <h3>When to consider something else</h3>
+        <p>For exploration of content, where users are browsing aimlessly for something interesting, infinite scroll is better suited than using pagination. Pagination is best when the user is trying to accomplish a goal, for instance trying to find a particular article from a list.</p>
+
+        <h3>How to use Pagination</h3>
+        <h4>Number of pages</h4>
+        <p>While the pagination component supports an infinite amount of pages, it is difficult to deliver a good user experience when the number of pages exceeds 10. In circumstances where it is difficult for a user to find what they are looking for within the first few search results, they may develop bad habits (such as finding what they are looking for on page 13, search result 5). To mitigate this, more advanced solutions than a single field search should be considered, such as categorization, labels, faceted search, etc</p>
```

Comment:
I think this reads a bit better.

```suggestion
        <p>While the pagination component supports an infinite amount of pages, it is difficult to deliver a good user experience when the number of pages exceeds 10. In circumstances where it is difficult for a user to find what they are looking for within the first few search results, they may develop bad habits (such as finding what they are looking for on page 13, search result 5). To mitigate this, more advanced solutions than a single search field should be considered, such as categorization, labels, faceted search, etc.</p>
```